src/test/java/org/isf/medicalsinventory/Tests.java

```diff
@@ -811,4 +828,86 @@ void testValidateMedicalInventory() throws Exception {
 		assertThat(medicalInventoryRows).hasSize(3);
 		medicalInventoryManager.validateMedicalInventoryRow(inventory, medicalInventoryRows);
 	}
+
+	@Test
+	void testValidateMedicalWardInventoryRow() throws Exception {
+		Ward ward = testWard.setup(false);
+		wardIoOperationRepository.saveAndFlush(ward);
+		MedicalInventory inventory = testMedicalWardInventory.setup(ward, false);
+		MedicalInventory savedInventory = medicalInventoryIoOperation.newMedicalInventory(inventory);
+		MedicalType medicalType = testMedicalType.setup(false);
+		medicalTypeIoOperationRepository.saveAndFlush(medicalType);
+		Medical medical = testMedical.setup(medicalType, false);
+		medicalsIoOperationRepository.saveAndFlush(medical);
+		Lot lot = testLot.setup(medical, false);
+		lotIoOperationRepository.saveAndFlush(lot);
+		MedicalInventoryRow medicalInventoryRow = testMedicalInventoryRow.setup(savedInventory, medical, lot, false);
+		int inventoryRowId = medicalInventoryRow.getId();
+		MedicalInventoryRow newMedicalInventoryRow = medicalInventoryRowManager.newMedicalInventoryRow(medicalInventoryRow);
+		assertThat(newMedicalInventoryRow).isNotNull();
+		List<MedicalInventoryRow> medicalInventoryRows = medicalInventoryRowManager.getMedicalInventoryRowByInventoryId(inventoryRowId);
+		assertThat(medicalInventoryRows).isNotEmpty();
+		assertThat(medicalInventoryRows).hasSize(1);
+		medicalInventoryManager.validateMedicalWardInventoryRow(savedInventory, medicalInventoryRows);
+		int inventoryId = inventory.getId();
+		inventory = medicalInventoryIoOperation.getInventoryById(inventoryId);
+		assertThat(inventory).isNotNull();
+	}
+
+	@Test
+	void testValidateMedicalWardInventory() throws Exception {
```

Comment:
there is not such method `validateMedicalWardInventory` in the manager but only `validateMadicalInventory` that BTW we forgot to test:
![image](https://github.com/user-attachments/assets/32112e67-4c98-4eb1-9b06-0b3e5ea713bf)

Are we going to use the same validation method for both Ward and Main Store inventories?

Please @ArnaudFofou verify also test coverage of your OP-1199.

Reply:
okay